### PR TITLE
Move integrations into separate section

### PR DESCRIPTION
**Pull request by kacperlukawski**
All the currently existing and future integrations are put into a separate docs section.

### CODE REVIEW

**Comment by kacperlukawski** [comment]
@azayarni @generall Should I ask somebody else for a review or simply merge it?

File: qdrant/v0.11.x/integrations.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Integrations
+weight: 52
+---
+Qdrant is a vector database, performing the
```
Comment (generall) [inline qdrant/v0.11.x/integrations.md:5]: typo?

Comment (kacperlukawski) [reply qdrant/v0.11.x/integrations.md]: Sorry about that, I had the intro already written, but somehow not committed. It's done.

[the remark this answers is not in the document]
Comment (kacperlukawski) [reply qdrant/v0.11.x/integrations.md]: Same as above. It's already put in place.

**Review by azayarni — APPROVED** [review]

**Review by generall — APPROVED** [review]

**Pull request merged**